R/wrappers.R

```diff
@@ -7,6 +7,7 @@ setcoalesce = function(...) .Call(Ccoalesce, list(...), TRUE)
 
 fifelse = function(test, yes, no, na=NA) .Call(CfifelseR, test, yes, no, na)
 fcase   = function(..., default=NA) .Call(CfcaseR, default, parent.frame(), as.list(substitute(list(...)))[-1L])
+topn    = function(vec, n=6L, decreasing=FALSE) .Call(CtopnR, vec, n, decreasing)
 
```

Comment:
Should the default be ```decreasing = TRUE```? When I think top I think highest values.  If it helps, ```dplyr::top_n``` uses the sign of n to determine whether the top n or bottom n are returned.

Reply:
I totally don't mind. The way I implemented it was to be in line with `order` and `sort`. `dplyr::top_n` does not work for me.

Reply:
I think `decreasing=TRUE` may be the more common use case, but practically speaking most sorting functions in R and elsewhere are doing `decreasing=FALSE` by default.

More practically, in other places in `data.table` we use lazy evaluation to allow `-` to mean `decreasing=TRUE` and switch the argument without requiring to negate the vector (should be efficient). In particular for character input `-` doesn't work for "flipping the sign".